centaur/src/main/resources/standardTestCases/docker_image_cache/docker_image_cache_true.wdl

```diff
@@ -2,16 +2,16 @@ version 1.0
 
 task check_if_docker_image_cache_disk_mounted {
   String google_docker_cache_disk_name = "google-docker-cache"
-  command {
+  command <<<
```

Comment:
TOL: Maybe let's update these in all 3 WDLs in this case

Reply:
Oh, this is leftover and no longer relevant. I think I should just revert TBH